### DISCUSSION TITLE
Error loading Calendar

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8001] [Archive]
Good afternoon,

All set correctly to work with the calendar, but when I browse between bookmarks and then go back to the calendar, it doesn't load the information, if you refresh the page, the information is loaded normally.

![image](https://user-images.githubusercontent.com/81326296/133129461-1d9c16e9-3450-4467-8ea5-e6188ca07598.png)
Refresh the Page - F5

![image](https://user-images.githubusercontent.com/81326296/133129372-2a281cff-5368-4aef-8a8d-b0f4c48223d6.png)

**Comment by rijkvanzanten** [comment]
Sounds like a bug to me. Mind opening an Issue? 🙂

**Reply by hugopvl** [reply]
ok :)
